patches/react-native-vision-camera+4.1.0.patch

```diff
@@ -0,0 +1,995 @@
++/Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/core/types/PreviewViewType.kt
++/Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/core/extensions/ListenableFuture+await.kt
++/Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/react/FpsSampleCollector.kt
+\ No newline at end of file
+diff --git a/node_modules/react-native-vision-camera/android/build/kotlin/compileDebugKotlin/local-state/build-history.bin b/node_modules/react-native-vision-camera/android/build/kotlin/compileDebugKotlin/local-state/build-history.bin
+new file mode 100644
+index 0000000..07da76b
+Binary files /dev/null and b/node_modules/react-native-vision-camera/android/build/kotlin/compileDebugKotlin/local-state/build-history.bin differ
+diff --git a/node_modules/react-native-vision-camera/android/build/outputs/logs/manifest-merger-debug-report.txt b/node_modules/react-native-vision-camera/android/build/outputs/logs/manifest-merger-debug-report.txt
+new file mode 100644
+index 0000000..ba32476
+--- /dev/null
++++ b/node_modules/react-native-vision-camera/android/build/outputs/logs/manifest-merger-debug-report.txt
+@@ -0,0 +1,16 @@
++-- Merging decision tree log ---
++manifest
++ADDED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml:1:1-3:12
++INJECTED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml:1:1-3:12
++	package
++		INJECTED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml
++	xmlns:android
++		ADDED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml:1:11-69
++uses-sdk
++INJECTED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml reason: use-sdk injection requested
++INJECTED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml
++INJECTED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml
++	android:targetSdkVersion
++		INJECTED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml
++	android:minSdkVersion
++		INJECTED from /Users/Matthew/git/edge-react-gui/node_modules/react-native-vision-camera/android/src/main/AndroidManifest.xml
+diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessors/VisionCameraProxy.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessors/VisionCameraProxy.kt
```

Comment:
Delete all the stuff above this line. It's all codegen stuff we don't need.

You can do the edit using VSCode, but beware of whitespace errors. Diffs are sensitive, and can get broken easily.

Editing with vim is safer, or just `rm node_modules/.yarn_integrity && yarn` to get a clean package, then re-do the edits below, and then re-run patch-package to get a clean diff.